src/protocols/bitcoin/components/TransferSendForm.vue

```diff
@@ -213,7 +217,14 @@ export default defineComponent({
           ).toNumber()))),
           time: 3540,
         },
-        { fee: new BigNumber(toBitcoin(mediumFee.toNumber())), time: 600 },
+        {
+          fee: new BigNumber(toBitcoin(
+            // Double the fee for the testnet to match relay fee.
+            // TODO: Revisit this along with fee calculation
+            mediumFee.toNumber() * (activeNetwork.value.type === NETWORK_TYPE_TESTNET ? 2.0 : 1.0),
```

Comment:
```suggestion
            mediumFee.toNumber() * (activeNetwork.value.type === NETWORK_TYPE_TESTNET ? 2 : 1),
```

Reply:
Specifically want to keep the fractions to indicate the decimal levels of the factor.

Reply:
I don't see this as a valid reason :) But we don't have time for such disputes so please move on.